.circleci/config.yml

```diff
@@ -1,182 +1,482 @@
-version: 2.1
+version: '2.1'
 
-executors:
-  # used for building the gem
-  ruby-latest:
-    resource_class: small
-    docker:
-      - image: cimg/ruby:3.1
-        environment:
-          BUNDLE_VERSION: '~> 2.3.4'
+meta:
+  ruby-version-matrix: &ruby-version-matrix
+    matrix:
+      parameters:
+        ruby-version:
```

Comment:
We now have one spot to update the ruby versions we wish to test with in a place that's easy to find.